megaraid.sh

```diff
@@ -31,11 +31,11 @@
 # if (metric['failed'] != 0) {
 #  return new AlarmStatus(CRITICAL, '#{failed} failed drive(s): #{report}');
 # }
-#
-# if (metric['prefail'] != 0) {
-#  return new AlarmStatus(WARNING, '#{prefail} prefail drive(s): #{report}');
-# }
-#
+
+if (metric['prefail'] != 0) {
+ return new AlarmStatus(WARNING, '#{prefail} prefail drive(s): #{report}');
+}
+
```

Comment:
This is a comment of example criteria, not the actual criteria that is used.  You'd have to modify where we're actually configuring the alarms for the change to be relevant.